Example/PY/flask/Readme.md

```diff
@@ -1,48 +1,35 @@
-﻿## Getting Started
-
-### Requirement
-
-Dependency|Version
----|----
-PYTHON | 3.5+
-GCC| GCC 5+
-cmake| 3.0+
-*inux|
-pinpoint| 1.8.0-RC1(thrift) <br> 2.0+(GRPC)
-
-### Installation
-#### install python virtual environment 
-
-https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/
-
-#### Install pinpoint-py module
-
-```
-$ cd PY/pinpoint-py-module
-$ sh setup_all_in_one.sh
-$ python setup.py install
-```
-#### integration pinpoint into your project
-
-> Steps
-
-1. Binding the preRequestPlugins
-
-``` python
-    @BaseHTTPRequestPlugins('SimpleWebServer',__name__)
-    def do_GET(self):
-```
-
-2. Binding the function/method you care
-
-```
-    @PinpointCommonPlugin('SimpleWebServer',__name__)
-    def getAgent(self,headers):
-```
-
-
-
-### TODO
-
-
-pip install --extra-index-url https://test.pypi.org/simple/ pinpointPy
\ No newline at end of file
+﻿# Integration Pinpoint Into Flask.
+
+
+## Integration pinpoint
+
+> Make sure pinpointPy module has been installed. ([How to Install pinpointPy module](DOC/PY/Readme.md))
+1. Copy [MyMiddleware.py](./MyMidWare.py) to your project. Add MyMidWare to your application.
```

Comment:
How about rename MyMiddleware to PinPointMiddleWare?
Don't forgot the  copyright !